includes/class-wc-siftscience-admin.php

```diff
@@ -168,7 +168,9 @@ private function output_settings_stats() {
 			$stats = json_decode( $stats , true );
 
 			foreach ( $stats as $outer_k => $outer_v ) {
-										
+				
+				$outer_k = '<span style="color:#00a0d2">' . str_replace('::', '</span>::', $outer_k);						
```

Comment:
Does this work? I'm not sure in PHP, but in many languages you're not allowed to modify the variable generated by a foreach loop.

Reply:
yes